src/logos.json

```diff
@@ -0,0 +1 @@
+{"brainly-mobile":"images/logos/brainly-mobile-6879551770.svg","brainly-plus-inverse":"images/logos/brainly-plus-inverse-c687c7219e.svg","brainly-plus-small":"images/logos/brainly-plus-small-57d2d0c2e9.svg","brainly-plus":"images/logos/brainly-plus-3f4fae7ad9.svg","brainly-tutoring-small":"images/logos/brainly-tutoring-small-02c148571b.svg","brainly-tutoring":"images/logos/brainly-tutoring-94373373cc.svg","brainly":"images/logos/brainly-5c4a769505.svg","eodev-mobile":"images/logos/eodev-mobile-bfdc46ee89.svg","eodev":"images/logos/eodev-a3f09bec5e.svg","nosdevoirs-mobile":"images/logos/nosdevoirs-mobile-2caead9ada.svg","nosdevoirs":"images/logos/nosdevoirs-5b0b83e2b7.svg","znanija-mobile":"images/logos/znanija-mobile-200611d052.svg","znanija-plus-inverse":"images/logos/znanija-plus-inverse-fdb3d35877.svg","znanija-plus-small":"images/logos/znanija-plus-small-495ddebd8c.svg","znanija-plus":"images/logos/znanija-plus-337423fa26.svg","znanija":"images/logos/znanija-63fd07a2eb.svg"}
```

Comment:
Could we please apply formatting in this file?

Reply:
Done